local/alertmanager/config/platform-config.yaml

```diff
@@ -48,4 +48,4 @@ receivers:
           url: '{{ .CommonAnnotations.link_url }}'
 
 templates:
-- /etc/alertmanager/ebsa-templates.tmpl
\ No newline at end of file
+- /etc/alertmanager/platform-templates.tmpl
```

Comment:
Tricky one this. Because this actually is an EBSA specific template matching the config they have on alert manager so we can see how our alert payload is being rendered.

Reply:
Renamed to `platform-e-template.tmpl`